workflow/cases/3dvar.yaml

```diff
@@ -3,6 +3,15 @@ case:
     SDATE: 2011-10-01t12:00:00
     EDATE: 2011-10-02t12:00:00
     godas_cyc: 1 # selection of godas cycle: 1(default)- 24h; 2 - 12h; 4 - 6h
+    resolution: Ocean025deg
```

Comment:
Considering to have the default case for now at least:
-  `Ocean1deg` 
-  `MOM6solo`